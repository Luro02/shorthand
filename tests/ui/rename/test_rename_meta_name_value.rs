use shorthand::ShortHand;

#[derive(ShortHand)]
struct Command {
    #[shorthand(rename = "example")]
    first: usize,
}

fn main() {}
