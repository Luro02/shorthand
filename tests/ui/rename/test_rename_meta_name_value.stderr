error: unexpected meta-item format `NameValue`, expected `List`
 --> $DIR/test_rename_meta_name_value.rs:5:17
  |
5 |     #[shorthand(rename = "example")]
  |                 ^^^^^^^^^^^^^^^^^^
